src/common/include/gudhi/Random.h

```diff
@@ -0,0 +1,56 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef RANDOM__H_
+#define RANDOM__H_
+
+#include <random>  // for std::random_device, std::mt19937, std::uniform_real_distribution
+#include <vector>
+#include <algorithm>  // for std::generate
+#include <cstddef>  // for std::size_t
+
+namespace Gudhi {
+  std::random_device rd;
```

Comment:
As a global object, it may cause trouble if you try to link 2 .o that both use gudhi.
You could mark it `inline` (and then also hide it as a static member of the class or in some subnamespace), or create a temporary one just where it is used.